wgpu-core/src/command/compute.rs

```diff
@@ -228,6 +235,14 @@ impl State {
     }
 }
 
+#[repr(packed)]
+#[allow(dead_code)]
+struct DispatchIndirectBuffer {
```

Comment:
These should be in wgpu-types so that wgpu users can use them too if they want to fill an indirect buffer on the cpu (useful for testing)